docs/design/layout.md

```diff
@@ -1 +1,427 @@
+        </P>
+      </Section>
+    </Pane.Column>
+  </Content>
+</div>
+```
+When designing the main content of a webpage, we recommend to follow these design rules:
+H1's are generally only used once per page, for the title. H2's should be used for the title of all sections on the page. H3's are used to display supplemental information to the title, such as a value. H4's should be used for sub-section titles. Sections should be divided with a 1 pixel thick divider. 
+
+If the main content is the only content on the page (there is no sidebar), the main content should not take up the full width of the available space. Generally the max-width value of the main content is set to 900px. This is to help maintain readability.
+
+Above is an example of a well-designed webpage which makes use of the Shared Component Library. It makes use of the [Pane.Column](/#!/PaneColumn) component, which inserts a divider and adds spacing between each of its child elements. We provide the spacing specifications of this example for your convenience.
+
+Margin specifications:
+  * Left:  30px
+  * Top:  30px
+  * Above and below dividers:  30px
+  * Between H2 and HelpText:  10px
+  * After paragraph:  30px
+  * After H4 sub-section:  30px
+
+## The Anchor Solar System
+
+```jsx
+<Spacing>
+  <Horizontal spacing='md' style={{alignItems: 'center'}}>
+    <H2>Phone Numbers</H2>
+    <Anchor icon="viewMore">
+      View all
+    </Anchor>
+    <Anchor icon="msExcel">
+      Export
+    </Anchor>
+    <Anchor icon="copy">
+      Copy to clipboard
+    </Anchor>
+  </Horizontal>
+</Spacing>
+```
+```jsx
+<Spacing>
+  <div style={{display: 'flex', flexDirection: 'row', justifyContent: 'flex-end', alignItems:'center'}}>
+    <div style={{marginRight:'0', marginLeft: '20px'}}>
+      <Anchor icon="copy" >
+        Copy to clipboard
+      </Anchor>
+    </div>
+    <div style={{marginRight:'0', marginLeft: '20px'}}>
+      <Anchor icon="msExcel" style={{marginRight:'0', marginLeft: '20px'}}>
+        Export
+      </Anchor>
+    </div>
+    <div style={{marginRight:'0', marginLeft: '20px'}}>
+      <Anchor icon="viewMore" style={{marginRight:'0', marginLeft: '20px'}}>
+        View all
+      </Anchor>
+    </div>
+    <Button style={{marginRight:'0', marginLeft: '20px'}}>Primary Medium</Button>
+  </div>
+</Spacing>
+```
+Toolbar anchors —usually represented by an icon and uppercase text that’s not underlined- act as actions for a given set of functionality. They generally appear next to a heading, but sometimes may appear next to a submit button depending on the context. Since pages can be subdivided into layout panes, sections, or blocks of content, it can be unclear where in the hierarchy to place these actions.
```

Comment:
Add space between – and words

Reply:
Technically em dashes don't have space on either side.